lib/samples/find_address_with_reverse_geocode/find_address_with_reverse_geocode_sample.dart

```diff
@@ -47,158 +51,74 @@ class _FindAddressWithReverseGeocodeSampleState
 
     final image = await ArcGISImage.fromAsset('assets/pin_circle_red.png');
     final pictureMarkerSymbol = PictureMarkerSymbol.withImage(image);
     pictureMarkerSymbol.width = 35;
     pictureMarkerSymbol.height = 35;
     pictureMarkerSymbol.offsetY = pictureMarkerSymbol.height / 2;
+
     _graphicsOverlay.renderer = SimpleRenderer(symbol: pictureMarkerSymbol);
-    _graphicsOverlay.graphics.add(_graphic);
     _mapViewController.graphicsOverlays.add(_graphicsOverlay);
 
     _mapViewController.arcGISMap = map;
+    _mapViewController.setViewpoint(_initialViewpoint);
 
-    _geocodeParameters.minScore = 75;
-    _geocodeParameters.resultAttributeNames
-        .addAll(['Place_addr', 'Match_addr']);
-
-    await Future.wait([
-      _worldLocatorTask.load(),
-      _sanDiegoLocatorTask.load(),
-    ]);
+    await _worldLocatorTask.load();
 
     setState(() => _ready = true);
   }
 
   void onTap(Offset localPosition) async {
-    final identifyGraphicsOverlayResult =
-        await _mapViewController.identifyGraphicsOverlay(
-      _graphicsOverlay,
-      screenPoint: localPosition,
-      tolerance: 5,
-    );
+    if (_graphicsOverlay.graphics.isNotEmpty) _graphicsOverlay.graphics.clear();
+
+    final mapTapPoint =
+        _mapViewController.screenToLocation(screen: localPosition);
+    if (mapTapPoint == null) return;
 
-    if (identifyGraphicsOverlayResult.graphics.isEmpty) return;
+    final normalizedTapPoint =
+        GeometryEngine.normalizeCentralMeridian(geometry: mapTapPoint);
+    displayTappedPoint(normalizedTapPoint!);
+
+    final reverseGeocodeParameters = ReverseGeocodeParameters();
+    reverseGeocodeParameters.maxResults = 1;
+    final reverseGeocodeResult = await _locatorTask.reverseGeocode(
+      location: normalizedTapPoint as ArcGISPoint,
+      parameters: reverseGeocodeParameters,
+    );
+    if (reverseGeocodeResult.isEmpty) return;
 
+    final firstResult = reverseGeocodeResult.first;
+    final address = firstResult.attributes['LongLabel'] as String;
     if (mounted) {
-      final graphic = identifyGraphicsOverlayResult.graphics.first;
-      final center = graphic.geometry!.extent.center;
-      final x = center.x.toStringAsFixed(3);
-      final y = center.y.toStringAsFixed(3);
-      final matchAddr = graphic.attributes['Match_addr'];
       showDialog(
         context: context,
         builder: (BuildContext context) {
-          return AlertDialog(content: Text('$matchAddr\n($x, $y)'));
+          return AlertDialog(content: Text(address));
         },
       );
     }
   }
 
-  void onServiceChanged(LocatorTask? locatorTask) {
-    if (locatorTask == null) return;
-
-    setState(() => _locatorTask = locatorTask);
-    Future.delayed(
-      const Duration(milliseconds: 250),
-      () => _scaffoldStateKey.currentState!.closeEndDrawer(),
+  void displayTappedPoint(Geometry mapPoint) {
+    final pointGraphic = Graphic(
+      geometry: mapPoint,
     );
```

Comment:
```suggestion
    final pointGraphic = Graphic(geometry: mapPoint);
```
Since this is just one property we can make it one line

Reply:
Yes I think you are right - this way is more concise and clearer